x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/hot_phase/hot_phase.js

```diff
@@ -64,194 +66,205 @@ class HotPhaseUi extends PureComponent {
-                      { value: 'd', text: intl.formatMessage({
-                        id: 'xpack.indexLifecycleMgmt.hotPhase.daysLabel',
-                        defaultMessage: 'days'
-                      }) },
-                      { value: 'h', text: intl.formatMessage({
-                        id: 'xpack.indexLifecycleMgmt.hotPhase.hoursLabel',
-                        defaultMessage: 'hours'
-                      }) },
-                    ]}
-                  />
-                </ErrableFormRow>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </Fragment>
-        ) : null}
-      </EuiDescribedFormGroup>
+                    errorKey={`${PHASE_ROLLOVER_MAX_AGE}`}
+                    isShowingErrors={isShowingErrors}
+                    errors={errors}
+                  >
+                    <EuiFieldNumber
+                      id={`${PHASE_HOT}-${PHASE_ROLLOVER_MAX_AGE}`}
+                      value={phaseData[PHASE_ROLLOVER_MAX_AGE]}
+                      onChange={async e => {
+                        await setPhaseData(PHASE_ROLLOVER_MAX_AGE, e.target.value);
+                      }}
+                      min={1}
+                    />
+                  </ErrableFormRow>
+                </EuiFlexItem>
+                <EuiFlexItem style={{ maxWidth: 188 }}>
+                  <ErrableFormRow
+                    id={`${PHASE_HOT}-${PHASE_ROLLOVER_MAX_AGE_UNITS}`}
+                    hasEmptyLabelSpace
+                    errorKey={PHASE_ROLLOVER_MAX_AGE_UNITS}
+                    isShowingErrors={isShowingErrors}
+                    errors={errors}
+                  >
+                    <EuiSelect
+                      aria-label={intl.formatMessage({
+                        id: 'xpack.indexLifecycleMgmt.hotPhase.maximumAgeUnitsAriaLabel',
+                        defaultMessage: 'Maximum age units'
+                      })}
+                      value={phaseData[PHASE_ROLLOVER_MAX_AGE_UNITS]}
+                      onChange={async e => {
+                        await setPhaseData(
+                          PHASE_ROLLOVER_MAX_AGE_UNITS,
+                          e.target.value
+                        );
+                      }}
+                      options={[
+                        { value: 'd', text: intl.formatMessage({
+                          id: 'xpack.indexLifecycleMgmt.hotPhase.daysLabel',
+                          defaultMessage: 'days'
+                        }) },
+                        { value: 'h', text: intl.formatMessage({
+                          id: 'xpack.indexLifecycleMgmt.hotPhase.hoursLabel',
+                          defaultMessage: 'hours'
+                        }) },
+                      ]}
+                    />
+                  </ErrableFormRow>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </Fragment>
+          ) : null}
+        </EuiDescribedFormGroup>
+        { phaseData[PHASE_ENABLED] ? (
```

Comment:
I don't see a way to disable this phase in the UI. Am I missing something? Is this condition necessary?